examples/gno.land/r/gov/dao/dao.gno

```diff
@@ -0,0 +1,60 @@
+package govdao
```

Comment:
Thank you for this confirmation. This PR is highly structural and allows people to focus on a specific point. 

For security concerns, we could start with a centralized DAO where the DAO is just a single public key. This would not yet be a DAO using votes, but at least it would not be an open-bar system.